views/home/home.css

```diff
@@ -0,0 +1,43 @@
+.badge-count-wrapper .menu-icon .badge {
+  height: 1.25rem;
+  width: 1.25rem;
+  padding: 0;
+  text-align: center;
+  right: 2rem;
+  top: 1rem;
+}
+.main-container .page .content {
+  width: 100%;
+}
+.banner-wrapper {
+  background-color: var(--white);
+}
+.banner {
+  position: relative;
+  display: block;
+}
+.banner-desc {
+  position: absolute;
+  top: 100px;
+  right: 100px;
```

Comment:
A suggestion: for future iterations, use variable for values that are use multiple times (like 2rem), also try to stick mostly to either px or rem , rest all looks good !

Reply:
sure, will convert them into variables.